Src/MainLib/InterceptFrame.cpp

```diff
@@ -903,13 +910,88 @@ bool InterceptFrame::IsBufferReadable(GLenum bufType, uint bufferCount) const
   return false;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//
+void InterceptFrame::GetBufferViewSize(GLenum bufType, uint bufferCount, GLint viewSize[4]) const
+{
+  viewSize[0] = 0;
+  viewSize[1] = 0;
+
+  if (this->fullscreen)
+  {
+    GLint fboId = 0;
+    if (extensionFBO) {
+      GLV.glGetIntegerv(GL_DRAW_FRAMEBUFFER_BINDING, &fboId);
+    }
+    if (fboId == 0)
+    {
+      // Default framebuffer, get view size from window
+#ifdef GLI_BUILD_WINDOWS
+      HDC hdc = GLW.wglGetCurrentDC();
+      HWND hwnd = WindowFromDC(hdc);
+      if (hwnd != nullptr)
+      {
+        RECT clientRect{};
+        if (GetClientRect(hwnd, &clientRect))
+        {
+          viewSize[2] = clientRect.right - clientRect.left;
+          viewSize[3] = clientRect.bottom - clientRect.top;
+          return;
+        }
+      }
+#endif
+    }
+    else if(iglGetFramebufferAttachmentParameteriv)
+    {
+      // FBO created by application
+      GLenum attachment = (bufType == GL_RGBA) ? GL_COLOR_ATTACHMENT0 + bufferCount :
+                          (bufType == GL_DEPTH_COMPONENT) ? GL_DEPTH_ATTACHMENT :
+                          GL_STENCIL_ATTACHMENT;
+      GLint objectType = GL_NONE;
+      iglGetFramebufferAttachmentParameteriv(GL_DRAW_FRAMEBUFFER, attachment, GL_FRAMEBUFFER_ATTACHMENT_OBJECT_TYPE, &objectType);
+      if (objectType == GL_TEXTURE && iglGetTexLevelParameteriv)
+      {
+        GLint mipLevel = 0;
+        iglGetFramebufferAttachmentParameteriv(GL_DRAW_FRAMEBUFFER, attachment, GL_FRAMEBUFFER_ATTACHMENT_TEXTURE_LEVEL, &mipLevel);
+
+        GLint textureName = 0;
+        iglGetFramebufferAttachmentParameteriv(GL_DRAW_FRAMEBUFFER, attachment, GL_FRAMEBUFFER_ATTACHMENT_OBJECT_NAME, &textureName);
+
+        GLint oldBinding;
+        GLV.glBindTexture(GL_TEXTURE_2D, textureName);
```

Comment:
Bind texture should happen after the old binding?

Reply:
Right, thanks for alertness :)